src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. For instance, if 'Roboto' fails to load, the browser should have an alternative font to display, such as 'Arial' or the default 'sans-serif'.